modules/kibana/src/main/java/org/elasticsearch/kibana/KibanaPlugin.java

```diff
@@ -38,6 +38,13 @@ public class KibanaPlugin extends Plugin implements SystemIndexPlugin {
         .setAllowedElasticProductOrigins(KIBANA_PRODUCT_ORIGIN)
         .build();
 
+    public static final SystemIndexDescriptor ONECHAT_INDEX_DESCRIPTOR = SystemIndexDescriptor.builder()
```

Comment:
From related issue:
> We should move as soon as we can to using .chat-* instead, to "decouple" it from pure Kibana system indices, and to have our own explicit system index pattern for this project.

Just for my understanding, what is the main motivation behind the need for decoupling?